llpc/lower/llpcSpirvLowerGlobal.cpp

```diff
@@ -1371,8 +1369,7 @@ void SpirvLowerGlobal::addCallInstForOutputExport(Value *outputValue, Constant *
 //
 // @param inOutTy : Type of this input/output member
 // @param addrSpace : Address space
-// @param indexOperands : Index operands
-// @param operandIdx : Index of the index operand in processing
+// @param indexOperands : Index operands to process
```

Comment:
Can you comment on the the minimum size required? For example, it seems like it wouldn't be valid to pass an empty array in here.

Reply:
The minimum size is 0. Empty indices means we have visited all of them. I will add a assert where we expect all indices are all handled.